rbac/management/role/model.py

```diff
@@ -72,7 +72,7 @@ class Access(models.Model):
     """An access object."""
 
     perm = models.TextField(null=False)
-    permission = models.TextField(null=False)
+    permission = models.TextField(null=False,  default="*:*:*")
```

Comment:
Looks like the linter is complaining about the whitespace here.
```suggestion
    permission = models.TextField(null=False, default="*:*:*")
```